app_base.htm

```diff
@@ -0,0 +1,55 @@
+<%inherit file="/fomo_app/templates/base.htm" />
+
+<%block name="catalog">
+
+    <div class="content row">
+
+        <!-- column -->
+        <div id="sidebar" class="col-2">
+            <ul class="text-center">
+                <a href="/catalog/index"><li class="sidebar ${'activeSide' if categoryId == 0 else ''}">All Products</li></a>
+                <%! from catalog import models as cmod %>
+                <% categories = cmod.Category.objects.all() %>
+                % for cat in categories:
+                    <a href="/catalog/index/${ cat.id }"><li class="sidebar ${'activeSide' if categoryId == cat.id else ''}">${ cat.name }</li></a>
+                % endfor

```

Comment:
Your way to create the li is really clever, but it is not the best practice to do this.

Reply:
I think this practice is ok. Albrecht said so at least. Thank you for your comment though.